contrib/tcpdump/template.py

```diff
@@ -0,0 +1,24 @@
+pkgname = "tcpdump"
+pkgver = "4.99.4"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = [
+    "--enable-smb",
+    "--with-crypto",
+]
+configure_gen = []
```

Comment:
you can also use build_style =cmake here instead and then you don't need to worry about this or the make_dir/cmd stuff (just make_check_target = "check")